cmd/start.go

```diff
@@ -16,6 +16,7 @@ const (
 	bearerTokenFlagName    = "bearer-token"
 	corsFlagName           = "cors-origin"
 	evaluatorFlagName      = "evaluator"
+	grpcCertPath           = "grpc-sync-cert-path"
```

Comment:
```suggestion
	grpcCertPathFlagName           = "grpc-sync-cert-path"
```
Just to keep all var names following the same pattern